### DISCUSSION TITLE
I cannot play audio files and video files in Network

### DISCUSSION THREAD

**Discussion by olsching** [TeamAmaze/AmazeFileManager#3925] [Q&A]
Hello, I have connected Amaze 3.8.4 to my network hard drive via smb.
However, I cannot play audio files and video files.
short display of an error message:
"... can not be opened"
Device: Sony compact xz1 (G8441) Android 9

(automatically translated)

**Comment by VishnuSanal** [comment]
@TranceLove 🙂
